pkg/cassandra/nodetool/nodetool.go

```diff
@@ -135,17 +124,6 @@ func (t *tool) Status() (NodeMap, error) {
 		)
 	}
 
-	if !mappedNodes.IsSuperset(leavingNodes.Union(joiningNodes).Union(movingNodes)) {
-		return nil, fmt.Errorf(
-			"mapped nodes must be a superset of leaving, joining and moving nodes. "+
-				"Leaving: %v, "+
-				"Joining: %v, "+
-				"Moving: %v, "+
-				"Mapped: %v",
-			leavingNodes, joiningNodes, movingNodes, mappedNodes,
-		)
-	}
-
```

Comment:
I'm confused - isn't this meant to be a part of another PR?

If yes and this is just stacked, it'd be great if we can stop stacking PRs too much as it makes it far harder to know which to review first. If we keep them separate, PRs that depend on [some other] PR will fail and/or we can add `/hold` and comment to inform a reviewer that "this PR depends on PR xyz being merged first"